JPF/mysql.skeleton.ipynb

```diff
@@ -152,36 +156,38 @@
     "    return dtypedict\n",
     "\n",
     "\n",
-    "def hpd_csv2sql(description, input_csv_file, sep_char, output_pickle,\\\n",
+    "def hpd_csv2sql(description, input_csv_url, sep_char,\\\n",
     "            table_name, dtype_dict, load_pickle, \\\n",
-    "                input_pickle, db_action, truncate_columns, date_time_columns,\\\n",
+    "                pickle_file, db_action, truncate_columns, date_time_columns,\\\n",
     "               chunk_size, keep_cols, max_col_len=255):\n",
-    "\n",
+    "    \n",
     "    log.info(\"Beginning {} Import {}\".format(description,datetime.datetime.now()))\n",
     "    \n",
-    "    if os.path.isfile(input_pickle):\n",
-    "        log.info(\"Flagged load of PICKLE: {} = True\".format(input_pickle))\n",
+    "    if load_pickle: #IF FLAGGED TO LOAD PICKLE AS TRUE\n",
+    "        log.info(\"Flagged load of PICKLE: {} = True\".format(pickle_file))\n",
     "        \n",
-    "        with open(input_pickle, 'r') as picklefile:\n",
-    "            log.info(\"Begin OPEN {} Pickle: {}\".format(input_pickle, datetime.datetime.now()))\n",
-    "            log.info(\"Great we have a pickle file...Loading from {}\".format(input_pickle))\n",
+    "        with open(pickle_file, 'r') as picklefile:\n",
```

Comment:
Right. Is there a way to pass flags or parameters into ipython notebook cells?

Ideally, you'd set up the logic to default to using what's on disk if exists, but override the behaviour if a flag is set.